02/4_extern_keyword/external_file.c

```diff
@@ -0,0 +1,8 @@
+#include "external_file.h"
+
+int g_extern_var = 0;

```

Comment:
let's add another global variable but this time don't use extern keyword. Comment this code out. In the codes slides, we can explain that if you uncomment this global variable, you will get compile error so try it out yourself